src/App.tsx

```diff
@@ -40,11 +47,14 @@ function App() {
       children: [
         { index: true, element: <Landing /> },
         { path:'explore', element: <Explore /> },
-        { path:'explore/:startData/:endDate/:persons', element: <Explore /> },
+        { path:'explore/:startDate/:endDate/:persons', element: <Explore /> },
```

Comment:
Don't forget to update this to be search params instead of different paths for strtDate and endDate and Persons.